src/definition/accessors/IJobBuilder.ts

```diff
@@ -0,0 +1,23 @@
+import { RecurrenceBuilder } from 'later';
+
+import { IJob, IJobResult, Schedule } from '../scheduler';
+import { IHttp } from './IHttp';
+import { IModify } from './IModify';
+import { IPersistence } from './IPersistence';
+import { IRead } from './IRead';
+
+export interface IJobBuilder {
+    setName(name: string): IJobBuilder;
+    getName(): string;
+    setDescription(description: string): IJobBuilder;
+    getDescription(): string;
+    setData(data: { [key: string]: any }): IJobBuilder;
+    getData(): { [key: string]: any };
+    parseTextSchedule(textToParse: string): IJobBuilder;
+    parseCronSchedule(cronToParse: string, includesSeconds?: boolean): IJobBuilder;
+    getRecurrenceScheduleBuilder(): RecurrenceBuilder;
+    setSchedule(schedule: Schedule): IJobBuilder;
+    getSchedule(): Schedule;
+    setExecutor(executor: (job: IJob, read: IRead, modify: IModify, http: IHttp, persis: IPersistence) => Promise<IJobResult>): IJobBuilder;
```

Comment:
The builder is one way of several which a job can be created. The other primary way is to implement the `IJob` interface and then handle everything yourself. So yes, it can have it's own entity as scheduling a job accepts the `IJob` interface.